launch.py

```diff
@@ -25,10 +25,11 @@ def env_defined(key):
 # Install Arma
 
 steamcmd = ["/steamcmd/steamcmd.sh"]
-steamcmd.extend(["+login", os.environ["STEAM_USER"], os.environ["STEAM_PASSWORD"]])
+steamcmd.extend(["+login", os.environ["STEAM_USER"],
+                os.environ["STEAM_PASSWORD"]])
```

Comment:
If you revert these changes here it'll be good to merge in

Reply:
Should be done